x-pack/plugins/ml/public/file_datavisualizer/components/file_datavisualizer_view/file_datavisualizer_view.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React, {
+  Component,
+} from 'react';
+
+import {
+  EuiFilePicker,
+  EuiSpacer,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+
+import { ml } from 'plugins/ml/services/ml_api_service';
```

Comment:
Should you use a relative path here to avoid errors with future Jest tests on this file?